cocos/2d/CCFontAtlasCache.cpp

```diff
@@ -88,22 +88,53 @@ FontAtlas* FontAtlasCache::getFontAtlasTTF(const _ttfConfig* config)
     return nullptr;
 }
 
-FontAtlas* FontAtlasCache::getFontAtlasFNT(const std::string& fontFileName, const Vec2& imageOffset /* = Vec2::ZERO */)
+FontAtlas* FontAtlasCache::getFontAtlasFNT(const std::string& fontFileName)
 {
-    auto realFontFilename = FileUtils::getInstance()->getNewFilename(fontFileName);  // resolves real file path, to prevent storing multiple atlases for the same file.
+    return getFontAtlasFNT(fontFileName, Rect::ZERO, false);
+}
+
+FontAtlas* FontAtlasCache::getFontAtlasFNT(const std::string& fontFileName, const std::string& subTextureKey)
+{
+    const auto realFontFilename = FileUtils::getInstance()->getNewFilename(fontFileName);  // resolves real file path, to prevent storing multiple atlases for the same file.
```

Comment:
I think should use `FileUtils::getInstance()->fullPathForFilename` instead.

Reply:
I don't know about that code.  That is from the current implementation, I never touched it.  Are you sure changing it would work?

Reply:
Oh, i thought it was added by you. Ok, let's keep it.